### PR TITLE
testing/py3-cairosvg: upgrade to 2.2.1; remove unsupported python2

### PR DESCRIPTION
**Pull request by stwa**
As stated on the NEWS site, python2 support is dropped since 2.0.0:

https://github.com/Kozea/CairoSVG/blob/master/NEWS.rst#version-200-released-on-2016-11-24

### CODE REVIEW

File: testing/py3-cairosvg/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Maintainer: William Pitcock <nenolod@dereferenced.org>
+pkgname=py3-cairosvg
+_pkgname=CairoSVG
+pkgver=2.2.1
+pkgrel=0
+pkgdesc="Cairo-based SVG renderer for python"
+url="http://cairosvg.org"
+arch="noarch"
+license="LGPL-3.0-or-later"
+options="!check"
```
Comment (andypost) [inline testing/py3-cairosvg/APKBUILD]: needs to explain why `check()` disabled

File: testing/py3-cairosvg/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Contributor: William Pitcock <nenolod@dereferenced.org>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: William Pitcock <nenolod@dereferenced.org>
```
Comment (andypost) [inline testing/py3-cairosvg/APKBUILD]: Makes sense to get approval from @kaniini

File: testing/py3-cairosvg/APKBUILD (hunk 1 above)
Comment (stwa) [reply testing/py3-cairosvg/APKBUILD]: I should have marked this as a WIP, I'm still working on getting checks right (missing dependencies).

**Comment by stwa** [comment]
Depends on https://github.com/alpinelinux/aports/pull/5679 and https://github.com/alpinelinux/aports/pull/5680.

**Comment by Ikke** [comment]
py3-cairosvg fails to build:

> ValueError: Only strings are accepted for the license field, files are not accepted

Could you also rebase this to fix conflicts and get drone CI?

**Pull request closed without merging**